domains/william2sober.json

```
@@ -0,0 +1,14 @@
+{
+  "description": "Documentation website for is-a.dev",
+  "repo": "https://github.com/william2sober/is-a-dev",
+  "owner": {
+    "username": "william2sober.is-a-dev",
+    "email": "willliam2sober@gmail.com"
+  },
+  "record": {
+    "A": [
+      76.76.21.21
```

should be double-quoted, as @DevMatei said
```suggestion
      "76.76.21.21"
```

---

Try now